README.md

```diff
@@ -5,14 +5,23 @@ A multiplayer [Chidiya Uddi](https://dsource.in/resource/indian-games/indoor-gam
 # Contribution Rules
 
 1. Simple: fork the repo, make meaningful and valuable code updates, create a pr.
-2. If you make GPT write all the code for your PRs, then _aandi mandi sandi..._ applies!
```

Comment:
Please put that back, it's fine if you use GPT for some parts but let this be here anyway, for the jokes maybe!